src/content/docs/zh-cn/reference/errors/unknown-compiler-error.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Unknown compiler error.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> 未知的编译器错误。 (E01000)
+
+## 哪里出了问题？
+Astro 在编译文件时遇到了未知错误。在大多数情况下，这不是你的错，而是编译器的问题。
+
+如果现有的 issue 中没有提到这个问题，请[创建一个 issue](https://astro.build/issues/compiler)。
```

Comment:
The compiler issues url returned 404 page.

I create a pr in English page.
 https://github.com/withastro/docs/pull/2838

Reply:
It fixed.